scheduler-worker/build.gradle

```diff
@@ -0,0 +1,136 @@
+plugins {
+  id 'java'
+  id 'application'
+  id 'com.github.node-gradle.node' version '3.5.0'
+}
+
+java {
+  toolchain {
+    languageVersion = JavaLanguageVersion.of(19)
+  }
+
+  withJavadocJar()
+  withSourcesJar()
+}
+
+node {
+  nodeProjectDir = file("${project.projectDir.toPath().resolve("scheduling-dsl-compiler")}")
+  version = '16.14.0'
+  npmVersion = '8.5.0'
+  download = true
+}
+
+task deleteConstraintsTypescript(type: Delete) {
+  delete project.projectDir.toPath().resolve('scheduling-dsl-compiler/src/libs/constraints')
+}
+
+task copyConstraintsTypescript(type: Copy) {
+  dependsOn deleteConstraintsTypescript
+  dependsOn ':merlin-server:assembleConstraintsDSLCompiler'
+  from rootDir.toPath().resolve('merlin-server/constraints-dsl-compiler/src/libs')
+  into project.projectDir.toPath().resolve('scheduling-dsl-compiler/src/libs/constraints')
+}
```

Comment:
I think we ended up with two copies of the constraints API floating around, even before this PR. @mattdailis do you know why we copy into `scheduling-dsl-compiler/src/libs/constraints` but also have the files in `scheduling-dsl-compiler/src/libs`?

Reply:
Here's where I added that: https://github.com/NASA-AMMOS/aerie/pull/179#issuecomment-1138060052 

We give the user code runner a list of filenames - I'm not sure that we can provide it filenames in any sort of namespaced way (i.e. imitating directory structure). However, during development, I think it's helpful to keep the copied constraints files in their own directory so they don't get mixed up with scheduling eDSL files.

So, only the files in the `scheduling-dsl-compiler/src/libs/constraints` matter at runtime.

The imports in `scheduling-edsl-fluent-api.ts` are written as though the constraints files were in the same directory (since they will appear to be at runtime). However, at build time, the typescript compiler complains about missing imports, so the similarly named files in `scheduling-dsl-compiler/src/libs/` simply import and re-export the copied files in `constraints`.

Happy to entertain alternative solutions - this is just what I came up with at the time to satisfy all of my constraints (no pun intended).

Reply:
Is this due to a limitation of the aerie user code runner, not letting us put the files in "virtual" directories?

Reply:
@pcrosemurgy this is not a blocker for merging

Reply:
Got it, marking this as "resolved" but it sounds like we should continue this conversation on this PR after it's closed, through Slack, or through GH discussions. :)